build_tools/benchmarks/comparisons/setup_desktop.sh

```diff
@@ -66,7 +77,7 @@ for i in $(ls ${ROOT_DIR}/models/tflite/); do
   MODEL_NAME=$(basename $i .tflite)
   echo "Processing ${MODEL_NAME} ..."
 
-  ${IREE_IMPORT_TFLITE_PATH} "${TFLITE_MODEL_DIR}/${MODEL_NAME}.tflite" -o "${IREE_MODEL_DIR}/${MODEL_NAME}.mlir"
+  ${IREE_IMPORT_TFLITE_PATH} "${TFLITE_MODEL_DIR}/${MODEL_NAME}.tflite" -o "${IREE_MODEL_DIR}/${MODEL_NAME}.mlir" || true
```

Comment:
Do we expect the model import will fail in some cases?

Reply:
Yes one of the models are actually failing (the ASR one). Since it takes a long time to go through all of the models and compile the different configs, we don't want the script to exit on failure of any single model.